### ISSUE TITLE
Latest Deno release (1.29.0) breaks ts-morph

### ISSUE THREAD

**Issue by itsdouges** [dsherret/ts-simple-ast#1372]
**Describe the bug**

Version: 17.0.1

**To Reproduce**

During initialization of a Project in 1.29 I get this error:

```
error: Uncaught (in promise) NotFound: No such file or directory (os error 2), stat '/Users/douges/projects/FAZE/node_modules/@typescript/lib-es2019'
```

In 1.28.3 I don't get it. I'll need to do some digging to understand what's happening, not sure why it's even looking in node modules.

```ts
import { Project } from 'https://deno.land/x/ts_morph@17.0.1/mod.ts';

const project = new Project();
```

**Expected behavior**

No error.

**Comment by bartlomieju** [comment]
Can you provide the command you use to run this program? I can't reproduce it locally on 1.29.0.

**Comment by dsherret** [comment]
I found a way to reproduce it. It seems like a deno bug. We're looking into it.

**Comment by dsherret** [comment]
Should be fixed in 1.29.1
